src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleLikelihoodMatrixMapper.java

```diff
@@ -58,7 +55,7 @@ public double get(final int alleleIndex, final int evidenceIndex) {
             }
 
             @Override
-            public int indexOfAllele(final A allele) {
+            public int indexOfAllele(final Allele allele) {
```

Comment:
This change is totally legal (e.g. see Map.containsKey) and fixes supress-warning issues in other places. Problem is that it cascaded to all descendants and that. is the reason this PR touches so many files.